CMakeLists.txt

```diff
@@ -231,13 +231,27 @@ include(CheckCXXCompilerFlag)
 add_custom_target(
     yosys ALL 
     COMMAND $(MAKE) config-gcc
-    COMMAND $(MAKE) 
+    COMMAND make clean
```

Comment:
Why make clean? This will force every compile of OpenFPGA to fully recompile yosys instead of just the changes.  This adds significant time to the compile.

Reply:
yes, this is correct, removed with https://github.com/lnis-uofu/OpenFPGA/pull/408/commits/a823c3e143ce6d67dd47d820b2eba29e498760b2